sway/sway.5.txt

```diff
@@ -94,6 +94,31 @@ Commands
 	workspace (or current workspace), and _current_ changes gaps for the current
 	view or workspace.
 
+**input** <identifier> tap <enabled|disabled>:
+	Enables or disables tap for specified input device.
+
+**input** <identifier> drag_lock <enabled|disabled>:
+	Enables or disables drag lock for specified input device.
+
+**input** <identifier> dwt <enabled|disabled>:
+	Enables or disables disable-while-typing for the specified input device.
+
+**input** <identifier> events <enable|disabled>:
+	Enables or disables send_events for specified input device.
+	(Disabling  send_events disables the input device)
+
+**input** <identifier> click_method <none|button_areas|clickfinger>:
+	Changes the click method for the specified device.
+
+**input** <identifier> middle_emulation <enabled|disabled>:
+	Enables or disables middle click emulation.
+
+**input** <identifier> scroll_method <none|two_finger|edge|on_button_down>:
+	Changes the scroll method for the specified input device.
+
+**input** <identifier> natural_scroll <enabled|disabled>:
+	Enables or disables natural scrolling for the specified input device.
+
```

Comment:
I intentionally didn't mention defaults because I don't know what they are with libinput. My understanding is that each device or type of device might have different defaults. That's also why I provide explicit enable/disable for each configuration option.